res/skins/QMLDemo/EffectSlot.qml

```diff
@@ -109,17 +108,17 @@ Item {
             id: parameter
 
             property int number: index + 1
-            property string name: shortName
+            property string label: shortName ? shortName : name
```

Comment:
can you add a comment that indicates that nullish coalescing (so `shortName ?? name`) will also work for Qt5.15?